controllers/storageautoscaler/storageautoscaler_controller_test.go

```diff
@@ -0,0 +1,27 @@
+package storageautoscaler
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCheckIfScalingNotRequired(t *testing.T) {
```

Comment:
This test only verifies where scaling is required. Some tests for not required would also be helpful to test edge conditions.

Reply:
Added